[2025-02-06T20:18:26.937+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T20:18:26.966+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-02-06T20:18:21.433135+00:00 [queued]>
[2025-02-06T20:18:26.977+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-02-06T20:18:21.433135+00:00 [queued]>
[2025-02-06T20:18:26.978+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-06T20:18:26.995+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_file_in_directory> on 2025-02-06 20:18:21.433135+00:00
[2025-02-06T20:18:27.003+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=6755) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-06T20:18:27.004+0000] {standard_task_runner.py:72} INFO - Started process 6757 to run task
[2025-02-06T20:18:27.005+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_file_in_directory', 'manual__2025-02-06T20:18:21.433135+00:00', '--job-id', '681', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmpk1mflf30']
[2025-02-06T20:18:27.006+0000] {standard_task_runner.py:105} INFO - Job 681: Subtask check_file_in_directory
[2025-02-06T20:18:27.057+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_file_in_directory manual__2025-02-06T20:18:21.433135+00:00 [running]> on host 003b7fbea265
[2025-02-06T20:18:27.348+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_file_in_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T20:18:21.433135+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-06T20:18:21.433135+00:00'
[2025-02-06T20:18:27.349+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-06T20:18:27.350+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-06T20:18:27.351+0000] {logging_mixin.py:190} INFO - Current task name:check_file_in_directory state:running start_date:2025-02-06 20:18:26.967611+00:00
[2025-02-06T20:18:27.351+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-02-06T20:18:27.352+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T20:18:27.353+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-06T20:18:27.357+0000] {python.py:240} INFO - Done. Returned value was: verify_file_reliability
[2025-02-06T20:18:27.358+0000] {branch.py:38} INFO - Branch into verify_file_reliability
[2025-02-06T20:18:27.358+0000] {skipmixin.py:230} INFO - Following branch ('verify_file_reliability',)
[2025-02-06T20:18:27.371+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-06T20:18:27.398+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T20:18:27.398+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_file_in_directory, run_id=manual__2025-02-06T20:18:21.433135+00:00, execution_date=20250206T201821, start_date=20250206T201826, end_date=20250206T201827
[2025-02-06T20:18:27.409+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-06T20:18:27.410+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-06T20:18:27.411+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-02-06 20:18:21.448512+00:00
[2025-02-06T20:18:27.411+0000] {logging_mixin.py:190} INFO - Task hostname:003b7fbea265 operator:BranchPythonOperator
[2025-02-06T20:18:27.461+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-06T20:18:27.489+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-06T20:18:27.492+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
